### PR TITLE
feat(cli): rename pkh option into address

### PR DESCRIPTION
**Pull request by bertux**
first draft for https://github.com/witnet/witnet-rust/issues/1051
still to need to test how to add the aliasing because I've seen for the command but not yet for the options

### CODE REVIEW

**Comment by aesedepece** [comment]
So far so good!

**Comment by bertux** [comment]
I've compiled and tested on my node and it works for both --address and --pkh
To stay coherent I have also renamed getPkh into getAddress but I've added getPkh as alias so both work also on my node server.
I've not changed the RPC to avoid breaking change but it helped me to understand better how it is called by command line.

File: src/cli/node/with_node.rs [hunk 1]
```diff
@@ -178,12 +178,16 @@ pub enum Command {
         /// Socket address of the Witnet node to query.
         #[structopt(short = "n", long = "node")]
         node: Option<SocketAddr>,
-        /// Public key hash for which to get balance. If omitted, defaults to the node pkh.
-        #[structopt(long = "pkh")]
-        pkh: Option<String>,
+        /// Public address for which to get balance. If omitted, defaults to the node address.
```
Comment (aesedepece) [inline src/cli/node/with_node.rs:181]:
```suggestion
        /// Address for which to get balance. If omitted, defaults to the node address.
```

File: src/cli/node/with_node.rs [hunk 2]
```diff
@@ -196,11 +200,11 @@ pub enum Command {
         /// Socket address of the Witnet node to query.
         #[structopt(short = "n", long = "node")]
         node: Option<SocketAddr>,
-        /// Public key hash for which to get reputation. If omitted, defaults to the node pkh.
-        #[structopt(long = "pkh")]
-        pkh: Option<String>,
+        /// Public address for which to get reputation. If omitted, defaults to the node address.
```
Comment (aesedepece) [inline src/cli/node/with_node.rs:203]:
```suggestion
        /// Address for which to get reputation. If omitted, defaults to the node address.
```

File: src/cli/node/with_node.rs [hunk 3]
```diff
@@ -219,9 +223,9 @@ pub enum Command {
         /// Socket address of the Witnet node to query.
         #[structopt(short = "n", long = "node")]
         node: Option<SocketAddr>,
-        /// Public key hash of the destination
-        #[structopt(long = "pkh")]
-        pkh: String,
+        /// Public address of the destination
```
Comment (aesedepece) [inline src/cli/node/with_node.rs:226]:
```suggestion
        /// Address of the destination
```

**Pull request closed without merging**